libs/mgjson/Makefile

```diff
@@ -0,0 +1,55 @@
+#
+# Copyright (C) 2021 OpenWrt.org
+#
+# This is free software, licensed under the GNU General Public License v2.
+# See /LICENSE for more information.
+#
```

Comment:
You should look in the CONTRIBUTING.md file in this repository, which says:
> Provide an up-to-date Copyright notice or none. Copyright should not be assigned to OpenWrt unless you are explicitly requested by or working under contract to OpenWrt. Assigning a Copyright to yourself or organization you represent is acceptable.

Reply:
again, a question i asked in  many places in which  you'd expect a thorough explanation ... got nothing. I assumed the copyright was some how tied to the example makefile i used originally, provided by the OpenWrt build system. 

Is there a better resource to have these types of questions answered ?

As i said in my previous comment ... these are my first commits. I have done what i consider my due diligence.  I have read any and all documentation offered.  I have no problem making any changes you suggest. I have tested everything to the absolute best of my abilities. I have compile with the all the relevant flags that I know of for error checking and have no errors anywhere in compilation , gcc is happy, valgrind reports no errors no leaks. I have tested everything thoroughly inside of the openwrt build environment, (yes make menuconfig loads the packages with no complaints, again everything is as expected.  

I will make the changes np, in fact I welcome the review, as  I have been requesting a code review in the forums to try avoid this exact situation. Please just let me know what you want changed/fixed, and I will gladly make the changes. If you can find any issue what so ever with my sources, please do point them out and i will be happy to investigate. Please just don't talk down to me like I should know all of this ...when no one seems to have time to answer these questions where they should in fact be answered.

Reply:
the copyright has been removed, as per your request